app/assets/stylesheets/base/_forms.scss

```diff
@@ -72,7 +72,12 @@ input[type="file"] {
 }
 
 select {
+  background-color: white;
+  border: 1px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.